gef-ui/server/utils/displayItems.js

```diff
@@ -9,7 +9,7 @@ const exporterItems = (exporterUrl) => [
   {
```

Comment:
Can the filename be changed to display-items.js? Camel case can work ok on local but cause issues on Azure if its not referenced correctly